ament_index_cpp/src/get_package_share_directory.cpp

```diff
@@ -17,14 +17,17 @@
 #include <string>
 
 #include "ament_index_cpp/get_package_prefix.hpp"
+#include "rcpputils/filesystem_helper.hpp"
```

Comment:
nitpick: I think `#include "ament_index_cpp/get_package_prefix.hpp"` should be in its own block after the include from `rcpputils`, based on this:

https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

And the fact that we often add a blank line between "Other libraries' .h files." and "Your project's .h files.".